.github/policies/mention-owners.yml

```diff
@@ -406,6 +406,17 @@ configuration:
                   replyTemplate: >-
                     Tagging subscribers to this area: ${mentionees}
                   assignMentionees: False
+          - if:
+              - labelAdded:
+                  label: area-Extensions-AI
+            then:
+              - mentionUsers:
+                  mentionees:
+                    - dotnet/dotnet-extensions-ai
+                    - shyamnamboodiripad
```

Comment:
@gewarren it would be better to create separate subareas (and labels) for MEAI and MEAI.Evaluation.

For MEAI, the team is https://github.com/orgs/dotnet/teams/dotnet-extensions-ai (so what you have above should work). Tagging @luisquintanilla and @stephentoub to identify additional individual mentionees if needed.

For MEAI.Evaluation, the team is https://github.com/orgs/dotnet/teams/dotnet-extensions-ai-evaluation (@dotnet/dotnet-extensions-ai-evaluation). You could include myself and / or @peterwald as individual mentionees.